lustrefs-exporter/src/jobstats.rs

```diff
@@ -134,6 +134,9 @@ pub fn jobstats_stream<R: BufRead + std::marker::Send + 'static>(
     let x = tokio::task::spawn_blocking(move || {
         let mut state = State::Empty;
 
+        // Send a new line to avoid printing at the same level as the previous stats
+        _ = tx.blocking_send('\n'.to_compact_string());
```

Comment:
I'm wondering why we can't just send '\n' as char. I believe there is a reason, but I'm curious to learn why.

Reply:
We send [`compact_str`](https://docs.rs/compact_str/latest/compact_str/) as a profiling optimization over the channel (while carefully trying to keep strings within stack size).